src/pages/workspace/withPolicy.tsx

```diff
@@ -1,21 +1,22 @@
-import _ from 'underscore';
-import lodashGet from 'lodash/get';
-import React from 'react';
+import React, {ComponentType, ForwardedRef, RefAttributes, forwardRef} from 'react';
 import PropTypes from 'prop-types';
-import {withOnyx} from 'react-native-onyx';
+import {OnyxEntry, withOnyx} from 'react-native-onyx';
 import {useNavigationState} from '@react-navigation/native';
 import CONST from '../../CONST';
 import getComponentDisplayName from '../../libs/getComponentDisplayName';
 import * as Policy from '../../libs/actions/Policy';
 import ONYXKEYS from '../../ONYXKEYS';
 import policyMemberPropType from '../policyMemberPropType';
+import * as OnyxTypes from '../../types/onyx';
 
-/**
- * @param {Object} route
- * @returns {String}
- */
-function getPolicyIDFromRoute(route) {
-    return lodashGet(route, 'params.policyID', '');
+type PolicyRoute = {
+    params?: {
+        policyID: string;
+    };
+};
+
+function getPolicyIDFromRoute(route: PolicyRoute): string {
+    return route?.params?.policyID ?? '';
 }
 
 const policyPropTypes = {
```

Comment:
Since we're already leaving this for places where it is being used, maybe let's create an alternative written in TS for future use-cases once other files get migrated to Typescript?